examples/calibration/plot_calibration_multiclass.py

```diff
@@ -141,7 +141,7 @@ class of an instance (red: class 1, green: class 2, blue: class 3).
 calibrated_classifier = sig_clf.calibrated_classifiers_[0]
 prediction = np.vstack([calibrator.predict(this_p)
                         for calibrator, this_p in
-                        zip(calibrated_classifier.calibrators_, p.T)]).T
+                        zip(calibrated_classifier.calibrators, p.T)]).T
```

Comment:
hum.. that's bad because we changed something that could be argue to be part of the public API since 0.23. We took care of it for `calibrated_classifiers_` but not for this nested attribute...

@glemaitre would you mind splitting this PR in two to merge them independently?

Reply:
But even if we add the backward compat we should indeed also update the example to avoid triggering the `FutureWarning`.

Reply:
I will do that.